deps/rabbitmq_auth_backend_oauth2/src/jwks_http_app.erl

```diff
@@ -0,0 +1,22 @@
+-module(jwks_http_app).
```

Comment:
Doesn't feel great to pollute the primary namespace with this.. should it be in the tests folder instead?

Reply:
You should be able to, e.g. [this suite uses a Common Test data directory](https://github.com/rabbitmq/rabbitmq-server/tree/master/deps/rabbit/test/unit_config_value_encryption_SUITE_data) (but it loads a file app descriptor directly).

`test` should be on the source code path. One complication here is that the node that runs the tests — a Common Test runner node — is separate from RabbitMQ nodes that run the plugins.

Reply:
That should be fine, as long as they're in the same network namespace.